cmd/dlv/cmds/commands.go

```diff
@@ -347,6 +347,9 @@ func debugCmd(cmd *cobra.Command, args []string) {
 			fmt.Fprintf(os.Stderr, "%v\n", err)
 			return 1
 		}
+		if runtime.GOOS == "windows" {
+			debugname += ".exe"
```

Comment:
I fear this would break backwards compatiblity. I don't know if anyone is using the output option but if they are we'll break them if we start changing the name of the file they are passing. If the problem is with the default value of output why not just change that?

Reply:
Yes, sorry for this pr. 
Some infomation about the processor of `go build` in my mind is wrong, I have tested some cases on windows.

Have no idea about `LookPath`, but this pr can be closed